### ISSUE TITLE
Make it more clear how to deploy your own peer discovery servers

### ISSUE THREAD

**Issue by karissa** [datproject/docs#156]
http://github.com/datproject/discovery

**Comment by RangerMauve** [comment]
Yeah, this'd be really cool! Docker containers seem like a good idea. Maybe we can have a Dat namespace on docker cloud.

Another thing that'd be cool is some sort of one-click deploy button for a cloud provider like Digital Ocean
